### ISSUE TITLE
Problem with subtraction

### ISSUE THREAD

**Issue by mokilcde** [danigm/gnome-shell-calculator#1]
Hi!

I try the following in calc (only an example)
`=1.1-0.9`
The result is:
`=0.20000000000000007`

Using Gnome 43 with calc version 11

**Comment by danigm** [comment]
Yep, that's how javascript works https://jsfiddle.net/tfrdn756/1/, this is related to floating point arithmetic https://en.wikipedia.org/wiki/Floating-point_arithmetic.

This calculator just run the operation with the javascript interpreter and there's no precise math operations or anything like that.
